frontend/src/components/pages/Scheduling/index.tsx

```diff
@@ -50,43 +51,58 @@ const Scheduling = () => {
   switch (id) {
     case "get started scheduling":
       return (
-        <GetStarted
-          formValues={schedulingFormValues}
-          setForm={setSchedulingForm}
-          navigation={navigation}
-        />
+        <>
+          <Header />
```

Comment:
```suggestion
  
```

instead of passing Header everywhere here, we can pass it in directly in the `App.tsx` before the `<Switch/>`